packages/manager/apps/telecom/src/app/telecom/telephony/repayments/list/repayments-list.html

```diff
@@ -20,7 +20,7 @@
         data-searchable
         data-filterable
     >
-        <span data-ng-bind="::$row.calledNumber | tucPhoneNumber"></span>
+        <span data-ng-bind="::$row.calledNumber"></span>
```

Comment:
**suggestion:** Extra `span` element can be avoided by addding `data-property` to the  `oui-datagrid-column`.

Reply:
It's applied.